pkg/yurtctl/cmd/yurttest/kindinit/init.go

```diff
@@ -411,9 +411,9 @@ func (ki *Initializer) configureAddons() error {
 		return err
 	}
 
-	if err := ki.ConfigureKubeProxyAddon(); err != nil {
-		return err
-	}
+	//if err := ki.ConfigureKubeProxyAddon(); err != nil {
```

Comment:
I guess this function should be deleted.

Reply:
fixed